manual/techdoc.tex

```diff
@@ -519,10 +519,4 @@ \subsection{Frame images compression}
 
 The DXT1 compression used to reduce size of the images is a from-scratch implementation of the Extreme DXT Compression\footnote{\url{http://www.cauldron.sk/files/extreme_dxt_compression.pdf}} algorithm. The conversion of natural color index order to DXT1 order is performed on the server side, as it is an invariant operation, without dependencies on the input data.
 
-\subsection{Thread naming}
-
-Most operating systems don't have adequate support for giving threads arbitrary names. On such systems the \texttt{TRACY\_COLLECT\_THREAD\_NAMES} macro will be defined, which enables storage of thread names in a lock-free list. On subsequent thread name queries this list is used, instead of the system facilities.
-
-Even if the lock-free list is used, Tracy will also set the thread name using the OS functionality. These names can be then used by debuggers and other external tools.
```

Comment:
This is now always enabled, see 71b54dd48aeb0ef185d9ccc6a9dbb7c8107bb169. A reword to remove mention of the macro would be better.